packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -727,6 +760,33 @@ export class ApexDebug extends LoggingDebugSession {
       }
     } catch (error) {
       this.tryToParseSfError(response, error);
+      // telemetry for expired session or invalid org-isv-debugger-sid (authentication error)
+      if (error === undefined) {
+        this.sendEvent(
+          new Event(SEND_METRIC_EVENT, {
+            subject: nls.localize('isv_debugger_session_authentication_invalid'),
+            type: 'startIsvDebuggerAuthenticationInvalid'
+          } as Metric)
+        );
+      }
+      // telemetry for invalid org-isv-debugger-url
```

Comment:
This block covers another edge case where the `org-isv-debugger-sid` config variable is set to an invalid value.  This is also a value that is supposed to be set correctly during the bootstrap process.